### ISSUE TITLE
Configuration files?

### ISSUE THREAD

**Issue by LikeLakers2** [godotengine/godot-tests#6]
[`/benchmarks/README.md`](https://github.com/godotengine/godot-tests/blob/master/benchmarks/README.md) and [`/tests/README.md`](https://github.com/godotengine/godot-tests/blob/master/tests/README.md) both mention that configuration files of some sort should be included. However, no example configuration is given, nor is the format documented that these configuration files would need to be in.

Has that format been decided before? If so, it should be documented in those README files. Otherwise, I hope this issue could spark some discussion on what format the configs should use.

**Comment by Calinou** [comment]
> Otherwise, I hope this issue could spark some discussion on what format the configs should use.

Would the ConfigFile class work for this?

**Comment by LikeLakers2** [comment]
> Would the ConfigFile class work for this?

Of course, but I was meaning what kind of data (name? brief description? etc.) these configuration files would need to have, not what the file type should be. My apologies for not clarifying.

**Comment by fire** [comment]
Anything that is actionable two years later?

**Comment by LikeLakers2** [comment]
I would think so. As mentioned in the original comment, the README files don't really explain what these "Configuration files" are, or how they're formatted, or what they're supposed to be. While it seems now that both take in Godot projects, it should be clarified since "Configuration file" could still mean any number of things (project settings? a python script? etc.).

We should either clarify that, or we should remove the two README files and explain this in the root of the repository.
